examples/run.py

```diff
@@ -1,5 +1,8 @@
 import argparse
 
+from datasets import load_dataset
+from torchvision.datasets import CIFAR10
+
```

Comment:
```suggestion
```

Reply:
suggest to move these to the appropriate sections so it's more clear these are optional